wire-team-settings/.env.defaults

```diff
@@ -37,7 +37,7 @@ URL_SUPPORT_LEGAL_HOLD="https://support.wire.com/hc/articles/360002018278"
 URL_SUPPORT_NEW_DE="https://support.wire.com/hc/de/requests/new"
 URL_SUPPORT_NEW_EN="https://support.wire.com/hc/en-us/requests/new"
 URL_SUPPORT_NEW_FR="https://support.wire.com/hc/fr/requests/new"
-URL_SUPPORT_NEW="https://support.wire.com/hc/requests/new"
+URL_SUPPORT_NEW="https://wire.com/contact/sales/"
```

Comment:
I think that's not correct. In your case you want to contact sales, but in every other case you still want to contact the support.